pkg/ssh/forward.go

```diff
@@ -77,6 +79,7 @@ func (f *forward) start(ctx context.Context) {
 			}
 
 			log.Infof("%s -> failed to accept connection: %v", f.String(), err)
+			<-tick.C
```

Comment:
If we've only seen this on the reverse, I think we should only do it there for now.

Reply:
Loops without a delay scare me

Reply:
Ok